### PR TITLE
add comment for validation cleanup

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: cli/dcoscli/marathon/main.py [hunk 1]
```diff
@@ -656,6 +656,7 @@ def _clean_up_resource_definition(properties):
         if v:
             if k in ["apps", "groups"]:
                 clean_properties[k] = [_clean_up_resource_definition(v[0])]
+            # remove properties added by marathon
```
Comment (jsancio) [inline cli/dcoscli/marathon/main.py]: Please be more describe. Comments should be about `why`.

Comment (tamarrow) [inline cli/dcoscli/marathon/main.py]: How's `remove non-user specified properties for validation since they aren't included in json-schema`?

**Comment by tamarrow** [comment]
I updated the comment for now and filed jiras for marathon and the cli to use/have a marathon update schema endpoint.

**Comment by jsancio** [comment]
:shipit:

**Pull request merged**